src/main/g8/src/main/scala/com/lightbend/akka/http/sample/QuickstartServer.scala

```diff
@@ -0,0 +1,98 @@
+package com.lightbend.akka.http.sample
+
+import akka.actor.{ ActorRef, ActorSystem }
+import akka.pattern.ask
+import akka.util.Timeout
+
+import scala.concurrent.duration._
+import akka.stream.ActorMaterializer
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.Http.ServerBinding
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.{ ExceptionHandler, Route }
+import akka.http.scaladsl.server.directives.MethodDirectives.delete
+import akka.http.scaladsl.server.directives.MethodDirectives.get
+import akka.http.scaladsl.server.directives.MethodDirectives.post
+import akka.http.scaladsl.server.directives.RouteDirectives.complete
+import akka.http.scaladsl.server.directives.PathDirectives.path
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.io.StdIn
+import scala.util.{ Failure, Success }
+import com.lightbend.akka.http.sample.UserRegistryActor._
+
+//#main-class
+object QuickstartServer extends App with JsonSupport {
```

Comment:
I think I remember either @ktoso or @jrudolph saying that they preferred importing those implicits instead of making them available via extending `JsonSupport`.